pebbles/static/partials/welcome.html

```diff
@@ -41,5 +42,11 @@ <h2>Login</h2>
             can <a href="#/reset_password">reset it here</a>
         </div>
     </div>
-    </div>
+    <div class="row" configurable-show key="COURSE_REQUEST_FORM"
+    name="course_request">
+      <div class="col-md-6">
+      <hr>
+      If you want to host your own course using notebooks.csc.fi, please fill out the <configurable-href key="COURSE_REQUEST_FORM" name="Course Request Form"></configurable-href>
```

Comment:
I realized the 'notebooks.csc.fi' part needs to be replaced with the host name i.e. INSTALLATION NAME